mixing/models/mixllama/configuration_mixllama.py

```diff
@@ -0,0 +1,39 @@
+import transformers
+from transformers import LlamaConfig
+from transformers.utils import logging
+
+from packaging import version
+
+logger = logging.get_logger(__name__)
+
+class MixLlamaConfig(LlamaConfig):
```

Comment:
Is this the right repo (and path) to be defining these models architectures?

There's a (currently closed) PR to add the same code to fms here, and other PRs to add MOE models in FMS as well: https://github.com/foundation-model-stack/foundation-model-stack/pull/245/files

I think if we can get this into an FMS format it would be nice to have MOE models in fms. If it's going to be pure-HF it might also even make sense to add directly to HF.

Reply:
@nairbv This is quite different from training a MOE architecture. This involves mixing pre-trained models to simply *create* an MOE-type model. Then it can be minimally trained via freezing all but the routers (or even not trained) and inferenced out-of-the-box. So, it need not be clubbed with the MOE PR. As for the closed PR to the FMS repo, it is indeed the same, we are trying to find the right repo for this. 
We want to get this on BAM so the internal community can play with this first.